modules/multi-architecture-modify-machine-set-aws.adoc

```diff
@@ -4,26 +4,23 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="multi-architecture-modify-machine-set-aws_{context}"]
+= Adding a multi-architecture compute machine set to your AWS cluster
 
-= Adding an ARM64 compute machine set to your cluster
-
-To configure a cluster with multi-architecture compute machines, you must create a AWS ARM64 compute machine set. This adds ARM64 compute nodes to your cluster so that your cluster has multi-architecture compute machines.
+To configure an AWS cluster with multi-architecture compute machines, you must create an AWS 64-bit ARM or 64-bit x86 compute machine set.
```

Comment:
This incipit does not give a so meaningful information now that the secondary architecture can be any. Perhaps, something like 
> When your cluster relies on the multi-architecture payload, you can add nodes with diverse architectures. For example, you can add amd64 workers alongside other arm64 workers in a cluster using arm64 control plane machines. In this case, we define the secondary architecture as `amd64`.

Reply:
Updated as follows:

"After creating a multi-architecture cluster, you can add nodes with different architectures. For example, you can add 64-bit ARM compute machines to a cluster that uses 64-bit ARM control plane machines and already includes 64-bit x86 compute machines. In this case, 64-bit ARM is considered the secondary architecture."